src/main/java/org/opensearch/security/dlic/rest/validation/AbstractConfigurationValidator.java

```diff
@@ -297,4 +312,18 @@ public String getMessage() {
     protected final boolean hasParams() {
         return param != null && param.length > 0;
     }
-}
\ No newline at end of file
+
+    private boolean hasNullArrayElement(JsonNode node) {
+        for (JsonNode element: node) {
+            if(element.isNull()) {
+                if (node.isArray())
```

Comment:
while code style is not explicitly enforced it will be better to adhere to what is mostly used by the rest of files. `if` must have `{}` block with `{` on the same line as `if` and `}` on the same line as `else if (`. Please see my code example for the reference.

Reply:
Fixed